src/shared/seccomp-util.c

```diff
@@ -196,15 +214,100 @@ const SyscallFilterSet syscall_filter_sets[_SYSCALL_FILTER_SET_MAX] = {
                 "sys_debug_setcontext\0"
         },
         [SYSCALL_FILTER_SET_DEFAULT] = {
-                /* Default list */
+                /* Default list: the most basic of operations */
                 .name = "@default",
                 .value =
+                "clock_getres\0"
+                "clock_gettime\0"
+                "clock_nanosleep\0"
                 "execve\0"
                 "exit\0"
                 "exit_group\0"
                 "getrlimit\0"      /* make sure processes can query stack size and such */
+                "gettimeofday\0"
+                "nanosleep\0"
+                "pause\0"
                 "rt_sigreturn\0"
                 "sigreturn\0"
+                "time\0"
+        },
+        [SYSCALL_FILTER_SET_FILE_SYSTEM] = {
+                /* Basic file system access (reading file properties, opening/creating files and directories for
+                 * read+write) */
+                .name = "@file-system",
+                .value =
+                "access\0"
+                "chdir\0"
+                "close\0"
+                "creat\0"
+                "faccessat\0"
+                "fchdir\0"
+                "fcntl64\0"
+                "fcntl\0"
+                "fgetxattr\0"
+                "flistxattr\0"
+                "fstat64\0"
+                "fstat\0"
+                "fstatat64\0"
+                "fstatfs64\0"
+                "fstatfs\0"
+                "getcwd\0"
+                "getdents64\0"
+                "getdents\0"
+                "getxattr\0"
+                "inotify_add_watch\0"
+                "inotify_init1\0"
+                "inotify_rm_watch\0"
+                "lgetxattr\0"
+                "listxattr\0"
+                "llistxattr\0"
+                "lstat64\0"
+                "lstat\0"
+                "mkdir\0"
+                "mkdirat\0"
+                "mmap2\0"
+                "mmap\0"
+                "newfstatat\0"
+                "open\0"
+                "openat\0"
+                "readlink\0"
+                "readlinkat\0"
+                "stat64\0"
+                "stat\0"
+                "statfs\0"
+        },
+        [SYSCALL_FILTER_SET_FILE_SYSTEM_OPS] = {
+                /* More advanced file system operations */
```

Comment:
May be it will be better if it is defined as a set of syscalls that update metadata attributes etc of already created files or so ... ?! I mean a better wording or description.